src/rpcserver.cpp

```diff
@@ -193,11 +193,6 @@ string CRPCTable::help(string strCommand) const
             continue;
         if ((strCommand != "" || pcmd->category == "hidden") && strMethod != strCommand)
             continue;
-#ifdef ENABLE_WALLET
```

Comment:
This means that the wallet commands will be listed in help, even if the wallet is disabled. Is that on purpose?